src/app/business/page.tsx

```diff
@@ -0,0 +1,6 @@
+import { commonFetch } from '@/api/commonFetch';
```

Comment:
These pages doesn’t make much sense. It will be better if you just add `category` url parameter to the home page